README.md

```diff
@@ -12,52 +12,53 @@ You can protect the DM area by setting a password.
 
 ![alt text](https://apclary.github.io/dungeon-revealer/img/example_player_1.jpeg "Player's view")
 
-## Installation
+## Getting Started
 
-### Docker
+### Getting the app
 
-A docker image is kept up to date with master branch of this repository.
+The easiest way to use dungeon-revealer is to download the app from the [releases](https://github.com/apclary/dungeon-revealer/releases) page here on github. There is also a [docker image](https://hub.docker.com/r/apclary/dungeon-revealer) that is kept up to date with the releases in this repository.
 
-To create an instance run the following:
+#### Prebuilt app 
 
-`docker run -e DM_PASSWORD=<password> -p <PORT>:3000 -d apclary/dungeon-revealer`
+Download the app for your system from [here](https://github.com/apclary/dungeon-revealer/releases).
 
-Replace `<password>` with your chosen DM password, and `<PORT>` with your prefered port.
+Either double click the app or open a command prompt in the same directory as the app and run from there. 
+Running from the command prompt will present connection information and some debugging.
+Optionally, you may set a password for the dungeon master page by setting the environmental variable `DM_PASSWORD` when starting the app. e.g. for linux `DM_PASSWORD='password' ./dungeon-revealer-linux`
+
+
+
+#### Docker
+
+To create an instance, run the following:
+
+```
+docker pull apclary/dungeon-revealer:latest
```

Comment:
this command is not necessary, since `docker run` will automatically fetch the image

Reply:
I put it in because if you have pulled the image in the past, then it will use the local image tagged with `latest` rather than fetch it from the repository. See [this issue](https://github.com/moby/moby/issues/13331).